### DISCUSSION TITLE
Turn off Front Display

### DISCUSSION THREAD

**Discussion by shortybiscuit** [gopro/labs#693] [Feature Requests]
A QR to quickly turn the front display on/off would be nice. If I missed it - show me how.

**Comment by dnewman-gpsw** [comment]
You missed it, all the setting are listed here: https://gopro.github.io/labs/control/settings/

oF0 <- turns the front screen off.

**Reply by shortybiscuit** [reply]
Thanks! I did not see this list AT ALL.
